dist/index.js

```diff
@@ -1062,11 +1088,19 @@ function loadConfig() {
         if (!fs_1.default.existsSync(filename)) {
             core.setFailed(`Could not find pipeline at ${baseFolder}/${pipeline}`);
         }
+        core.info(`file parameters ${runtimeParametersFile}`);
+        core.info(`folder parameters ${folderName}`);
+        if (runtimeParametersFile) {
+            core.info("Reading runtime parameters file");
+            runtimeParametersFile = readParemetersFile(folderName, pipeline, runtimeParametersFile);
+            core.info(`file parameters update! ${runtimeParametersFile}`);
```

Comment:
Same, this one not needed.

Reply:
I added those lines for debugging I already deleted